### ISSUE TITLE
Ubuntu 22.04 Problem [libssl]

### ISSUE THREAD

**Issue by dariosena** [frectonz/sql-studio#36]
Hello, amazing project!

I tried locally and got the following error:

![image](https://github.com/frectonz/sql-studio/assets/348842/7b7929b2-bbec-4510-ac8d-cbacab6a341a)

For me the default is `libssl3`, and for security reasons I sticking to it. Do you have some info about? Is something that you saw before? Need some help?

UPDATE: Building locally, runs fine! The executable provided by the `install` command is probably targeting some old `debian, ubuntu (?)` with outdated `libssl`.

Thx!

**Comment by frectonz** [comment]
Yeah will fix this problem. This is problem only exists in `0.1.20` use `0.1.19` until it's fixed.

```bash
curl --proto '=https' --tlsv1.2 -LsSf https://github.com/frectonz/sql-studio/releases/download/0.1.19/sql-studio-installer.sh | sh
```

**Comment by frectonz** [comment]
Fixed in `0.1.21`
